### ISSUE TITLE
External Users cannot access Operation tile when Operator status = "Draft"

### ISSUE THREAD

**Issue by andrea-williams** [bcgov/cas-registration#591]
#### Description:

Operators that have been imported from other DBs will have their status set to "Draft" by default.

External users should not be allowed to add any new Operations or edit existing Operation data until they have reviewed and confirmed the imported Operator data.

See [Miro board](https://miro.com/app/board/uXjVN6uBj9Y=/?share_link_id=624499766165) for context on this workflow.

#### Acceptance Criteria:

Given I am an external user with access to an Operator
When my Operator's status = "Draft"
Then I CANNOT access the Operations tile on my dashboard without first navigating to the "My Operator" tile and confirming (or updating) the imported Operator data.

Given I am an external user with access to an Operator
When my Operator's status = "Draft" and I click on the "My Operator" tile in the dashboard
Then I can view the imported 

Given I am an external user with access to an Operator
When my Operator's status = "Approved" or "Pending"
Then I CAN access the Operations tile on my dashboard

#### Development Checklist:

- [x] routing is enforced based on Operator status and external user privileges
- [x] external users are notified in Dashboard that they must confirm their Operator data before proceeding with any Operations forms/data (see Hanna's comment in [this wireframe](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=2414%3A1571&mode=design&t=7fNiOK0sRqt4IUdu-1))
- [x] once the user has clicked the "Save and return to Dashboard" button, the Operator status is set to "Pending" (and any edits the user made are saved). (See [this wireframe](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=2401%3A331&mode=design&t=7fNiOK0sRqt4IUdu-1), paying attention to the changed button at the bottom of the form)
- [x] Meets the DOD

Notes: which tiles show is currently role-based and handled in /cas-registration/client/app/(authenticated)/dashboard/page.tsx. Can make a new json file with/without operations tile and conditionally show it in this case: case FrontEndRoles.INDUSTRY_USER:
            contentsModule = await import(
              "@/app/data/dashboard/industry_user.json"
            );

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by Dianadec** [comment]
Trying out a little estimation experiment... Team please add your estimates: https://planningpokeronline.com/zz1T5DgYXidDPQtTKck2/ 
@andrea-williams @marcellmueller @Sepehr-Sobhani @dleard @shon-button @BCerki

**Comment by hannavovk** [comment]
The third AC "operator status is set to 'Pending' after clicking Save and Return to Dashboard " does not seem to be working - Operations tile does not show up. Can someone look into what's going on there? @BCerki @andrea-williams 
Also, there is no indication that an action is required on the Operator tile. @dleard created a new ticket for this one - see #876

**Comment by BCerki** [comment]
> The third AC "operator status is set to 'Pending' after clicking Save and Return to Dashboard " does not seem to be working - Operations tile does not show up. Can someone look into what's going on there? @BCerki @andrea-williams Also, there is no indication that an action is required on the Operator tile. @dleard created a new ticket for this one - see #876

I'll look into the tile in a separate PR
